descriptors/ModuleDescriptor-template.json

```diff
@@ -232,13 +232,29 @@
       "id": "users",
       "version": "16.0"
     },
+    {
+      "id": "loan-policy-storage",
+      "version": "2.3"
+    },
     {
       "id": "loan-storage",
       "version": "7.1"
     },
     {
-      "id": "loan-policy-storage",
-      "version": "2.3"
+      "id": "organizations",
+      "version": "3.3"
+    },
+    {
+      "id": "po-lines",
```

Comment:
I think this should be "orders-storage.po-lines". We may also want to use the version from the latest release (12.2), rather than the one on master.